core/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -492,6 +493,7 @@ abstract class AppBase(
   def processEvents(): Unit = {
     processEventsInternal()
     postCompletion()
+    RuntimeInjector.insertMemoryMarker("Post processing events")
```

Comment:
Q: Why do we want to do GC before printing the `memoryInfo` here?

Reply:
Because freeMemory is supposed to be the free size in the heap in bytes.
In order to get that value, first, we should collect all the garbage.